### PR TITLE
chore(suite): setup ev certificate signing for windows app

### PR DESCRIPTION
**Pull request by vdovhanych**
This adds functionality to sign the Windows app using a new EV signing certificate.
This is done based on the [electron-builder documentation](https://www.electron.build/tutorials/code-signing-windows-apps-on-unix#signing-windows-app-on-maclinux-using-jsign) that suggests doing the signing this way if you want to do it on Linux/macOS 

Related with - #12767

### CODE REVIEW

File: packages/suite-desktop-core/scripts/sign-windows.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+exports.default = async function (configuration) {
```
Comment (komret) [inline packages/suite-desktop-core/scripts/sign-windows.ts:2]: Shouldn't be async? See Files changed for other linter warnings as well.

**Review by komret — CHANGES_REQUESTED** [review]

File: packages/suite-desktop-core/scripts/sign-windows.ts (hunk 1 above)
Comment (vdovhanych) [reply packages/suite-desktop-core/scripts/sign-windows.ts:2]: Yeah was just fixing it :) I took the script from electron documentation i guess its not really good one.. :D

**Review by komret — APPROVED** [review]

File: packages/suite-desktop-core/scripts/sign-windows.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    execSync(
+        `java -jar ../suite-desktop-core/scripts/jsign-6.0.jar --keystore ../suite-desktop-core/scripts/hardwareToken.cfg --storepass '${TOKEN_PASSWORD}' --storetype PKCS11 --tsaurl http://timestamp.digicert.com --alias "${CERTIFICATE_NAME}" "${configuration.path}"`,
+        {
+            stdio: 'inherit',
+        },
+    );
+}
+
+export default signWindows;
```
Comment (matejkriz) [inline packages/suite-desktop-core/scripts/sign-windows.ts]:
```suggestion
// eslint-disable-next-line import/no-default-export
export default signWindows;
```

**Comment by matejkriz** [comment]
@komret this needs to be cherry-picked later in release/24.6 (only, not in 24.5)

**Pull request merged**